bridge/client/app/app.module.ts

```diff
@@ -328,6 +325,7 @@ export function init_app(appLoadService: AppInitService): () => Promise<unknown>
     CommonModule,
     KtbHeatmapModule,
     KtbPipeModule,
+    KtbLoadingModule,
     KtbNotificationModule,
   ],
```

Comment:
May I ask you to sort the imports alphabetically? I forgot it in my PR, but we agreed on doing so, to reduce the risk of having to many conflicts while changing the same file in multiple PRs.